### DISCUSSION TITLE
Spawn reports frequent errors and some lsp services cannot be installed

### DISCUSSION THREAD

**Discussion by OpenTritium** [mason-org/mason.nvim#280] [Q&A] [answered]
report like this:
spawn: luarocks failed with exit code - and signal -. luarocks is not executable
spawn: unzip failed with exit code - and signal -. unzip is not executable

**Comment by wheregrassdreams** [comment, marked as answer]
just install what report mentioned that is not executable

`sudo apt install luarocks`
`sudo apt install unzip`
